lib/manageiq/environment.rb

```diff
@@ -21,7 +21,7 @@ def self.manageiq_plugin_update(plugin_root = nil, force_bundle_update: true)
       ensure_config_files
 
       puts "== Installing dependencies =="
-      setup_gemfile_lock if ci?
+
```

Comment:
alternatively, if we want to make a slow transition, we can run this unless we are running build version 2...

```suggest
setup_gemfile_lock if ci? && ENV["BUILD"] != "v2"
```